### ISSUE TITLE
Import Error from click.termui 

### ISSUE THREAD

**Issue by Tobi-De** [MousaZeidBaker/poetryup#38]
Hi, Honestly, I have no idea where this is coming from, I used the package a couple of days ago to update some dependencies and it worked fine. I'm currently using the latest 0.6.0 version, I tried to downgrade to 0.5.4 and 0.5.3 and same issue.  I think it happened during a poetrup upgrade, but I don't know when.

```sh
Traceback (most recent call last):
  File "/Users/tobi/Library/Caches/pypoetry/virtualenvs/fwellbeing-hsd0M_FB-py3.10/bin/poetryup", line 5, in <module>
    from poetryup.main import app
  File "/Users/tobi/Library/Caches/pypoetry/virtualenvs/fwellbeing-hsd0M_FB-py3.10/lib/python3.10/site-packages/poetryup/main.py", line 7, in <module>
    import typer
  File "/Users/tobi/Library/Caches/pypoetry/virtualenvs/fwellbeing-hsd0M_FB-py3.10/lib/python3.10/site-packages/typer/__init__.py", line 12, in <module>
    from click.termui import get_terminal_size as get_terminal_size
ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/Users/tobi/Library/Caches/pypoetry/virtualenvs/fwellbeing-hsd0M_FB-py3.10/lib/python3.10/site-packages/click/termui.py)

```

**Comment by MousaZeidBaker** [comment]
Hi @Tobi-De, thanks for reporting an issue. 

`poetryup` depends on [Typer](https://github.com/tiangolo/typer) which in turn depends on [click](https://github.com/pallets/click), and `click` has [removed `get_terminal_size`](https://github.com/pallets/click/pull/2130) in version `8.1.0`. An issue has been reported in `Typer`: https://github.com/tiangolo/typer/issues/377 and a fix seems to be around the corner. A temporary workaround is to lock `click` to `8.0.4`.

**Comment by Tobi-De** [comment]
> Hi @Tobi-De, thanks for reporting an issue.
> 
> `poetryup` depends on [Typer](https://github.com/tiangolo/typer) which in turn depends on [click](https://github.com/pallets/click), and `click` has [removed `get_terminal_size`](https://github.com/pallets/click/pull/2130) in version `8.1.0`. An issue has been reported in `Typer`: [tiangolo/typer#377](https://github.com/tiangolo/typer/issues/377) and a fix seems to be around the corner. A temporary workaround is to lock `click` to `8.0.4`.

Ok got it, thanks

**Comment by tiangolo** [comment]
I just released Typer `0.4.1` that should handle this. :rocket: :nerd_face:
